crates/sui-graphql-rpc/src/lib.rs

```diff
@@ -2,17 +2,20 @@
 // SPDX-License-Identifier: Apache-2.0
 
 pub mod commands;
+pub mod config;
 pub mod server;
 
 mod context_data;
 mod error;
 mod extensions;
+mod functional_group;
 mod types;
 
-use crate::types::query::Query;
 use async_graphql::*;
 use types::owner::ObjectOwner;
 
+use crate::types::query::Query;
```

Comment:
Did you reorder these mod and uses manually?

Reply:
I did, because `cargo fmt` won't break out imports by default, but if it introduces a new import, it tries to keep `crate`, `std` and other imports separate from each other, IIRC.